package.json

```diff
@@ -66,6 +68,7 @@
 		"prop-types": "^15.7.2",
 		"react": "^17.0.0",
 		"react-dom": "^17.0.0",
+		"regenerator-runtime": "^0.13.7",
```

Comment:
I had to add this and `babel-jest` to be able to run async test functions. It seems like normally this shouldn't be necessary, but I couldn't find another way. Would be good to look into that later.